addons/music/fnc_playMusic.sqf

```diff
@@ -0,0 +1,48 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_playMusic
+Description:
+    A function used to play music. 
+Parameters:
+    1: CLASS Class to play
+    2: INT Position to play from (default: 0)
+    3: BOOL allowed to interupt already playing music (default: true)
+Example:
+    (begin example)
+    _bool = ["LeadTrack01_F_Bootcamp", 30, true] call CBA_fnc_playMusic;
+    (end example)
+Returns:
+    BOOL
+Author:
+    Fishy
+---------------------------------------------------------------------------- */
+#include "script_component.hpp"
+private ["_config","_data","_canPlay","_duration","_return"];
+params [["_className",""],["_time",'name'],["_overWrite", true]];
+if (_className == "") exitWith {
+    WARNING("No class given"); 
+    false
+};
+
+_return = false;
+_canPlay = false;
+
+if (_overWrite) then {
+    _canPlay = true;
+} else {
+    if (!([true] call CBA_fnc_getMusicPlaying)) then {
+    _canPlay = true;
+    }; 
+};
+
+if (_canPlay) then {
```

Comment:
`if (!_canPlay) exitWith {_return}` or `if (!_canPlay) exitWith {false}`  might be nicer looking.